Contributing.md

```diff
@@ -14,6 +15,31 @@ replace `make tidy` with rustfmt as a medium-term goal.
 Having a strong test suite for a tool like this is essential. It is very easy
 to create regressions. Any tests you can add are very much appreciated.
 
+The tests can be run with `cargo test`. This does a number of things:
+* runs the unit tests for a number of internal functions;
+* makes sure that rustfmt run on every file in `./tests/source/` is equal to its
+  equivalent in `./tests/target/`;
+* confirms that the output of files in `./tests/target/` is equal to the input;
+* checks that rustfmt's code is idempotent under itself.
```

Comment:
And here could you mention that this ensures that all of rustfmt is rustfmt'ed, i.e., the project bootstraps